vendor/github.com/moby/moby/daemon/caps/utils_unix.go

```diff
@@ -90,36 +90,26 @@ func TweakCapabilities(basics, adds, drops []string) ([]string, error) {
 	// FIXME(tonistiigi): docker format is without CAP_ prefix, oci is with prefix
 	// Currently they are mixed in here. We should do conversion in one place.
 
-	// look for invalid cap in the drop list
+	// handle --cap-add=all
+	if inSlice(adds, "all") {
```

Comment:
this change is inside the vendored moby code - won't this prevent future upgrades?

Reply:
oh yep I totally missed this - serves me right for reivewing on a train